### DISCUSSION TITLE
Editing location data

### DISCUSSION THREAD

**Discussion by chris-stg** [code4recovery/12-step-meeting-list#1213] [Q&A]
Is there a way to edit locations directly via the UI? We are finding a handful of locations are resolving towns incorrectly and need to be updated. If not, what tables are are involved on the database?

**Comment by joshreisner** [comment]
hi @chris-stg - there's no way to change a location in the UI, but there is a way to override an address by editing your theme's functions.php: https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location

if you want we can help you form the code you'll need to add. which addresses are giving you trouble?

https://www.latlong.net/ can be a resource to find the correct latitude and longitude if google is not giving the correct response (we use the google geocoding api in the plugin)
